tests/io/v0_0_0/test_write.py

```diff
@@ -180,3 +181,54 @@ def test_checksum_on_write(self):
                 loaded = slicedimage.Reader.parse_doc(basename, baseurl)
 
                 loaded.tiles()[0]._load()
+
+    def test_write_tiff(self):
+        image = slicedimage.TileSet(
```

Comment:
nit: using kwargs to call these arguments would be clearer for users looking to this code as use-case examples.